### PR TITLE
Add ability to set guest name in lobby

### PR DESCRIPTION
**Pull request by ma12-co**
Signed-off-by: Marco Ambrosini <marcoambrosini@pm.me>

### CODE REVIEW

File: src/components/LobbyScreen.vue [hunk 1]
```diff
@@ -19,20 +19,28 @@
...
-		<div class="icon icon-lobby" />
-		<h2>{{ currentConversationName }}</h2>
-		<p>{{ message }}</p>
+	<div class="lobby">
+		<div class="lobby emptycontent">
+			<div class="icon icon-lobby" />
+			<h2>{{ currentConversationName }}</h2>
+			<p>{{ message }}</p>
+		</div>
+		<SetGuestUsername />
```
Comment (nickvergessen) [inline src/components/LobbyScreen.vue]: Missing check for guest?
Also normal users end up in the lobby

Comment (ma12-co) [reply src/components/LobbyScreen.vue]: Check for guest is in the parent component!

Comment (ma12-co) [reply src/components/LobbyScreen.vue]: feature/noid/show-selected-participants-in-new-group-creation

Comment (ma12-co) [reply src/components/LobbyScreen.vue]: Sorry you're right :)

**Comment by jancborchardt** [comment]
This was btw in the spec for the lobby ;)

> - You can set your name already before you join the meeting.
> Talk lobby

https://github.com/nextcloud/spreed/pull/1926#issuecomment-506962798

**Comment by ma12-co** [comment]
Wow, that looks so different than what we have now... I just copied the same component that we use in the sidebar for now, we did it together in January's contributor week @jancborchardt. Don't you think that it's better than the explicit input field??  

Peek 2020-03-27 11-19

**Comment by jancborchardt** [comment]
@ma12-co the idea was that _if_ the name is the default "Guest", you probably want to change it and we can directly show the input field. It wouldn’t need to show as input field if you already set a name. My only suggestions for that:
- We should call it "Your name" instead of "Display name" as that’s quite technical
- The input field could replace the current name and the edit button in place, especially

> Wow, that looks so different from what we have now...

Yeah – it’s rather bland at the moment. I’d still love for the mockup and spec to be followed a bit more:
- It looks much much better. We don’t necessarily need to have the dark background, but this standard blue header and the white content just looks so basic.
- Gives you feedback as to who you will be talking to – just like e.g. Hangouts does before joining
- Shows your own camera in case it’s a video call – also just like Hangouts
- Unclear what "waiting in the lobby" means. "Waiting for moderators to start the call" is clear and you know who to talk to if it does not start.
- Some other points in the spec as well – give it a read at https://github.com/nextcloud/spreed/pull/1926#issuecomment-506962798

**Comment by ma12-co** [comment]
Ok created an issue from your comment

**Pull request merged**